### ISSUE TITLE
AT: confusing message when uploading a theme

### ISSUE THREAD

**Issue by rickybanister** [Automattic/wp-calypso#39256]
While doing my support rotation a customer wrote in confused by this message:

> This feature isn't (yet) compatible with theme uploads and will be disabled:
> WordPress.com Subdomains: Your subdomain will change from abc123.wordpress.com to abc123.wpcomstaging.com.
> Questions?: Contact support for help.

They had just completed an inbound domain transfer and thought that message was telling them that their domain would prevent them from uploading a theme to their site. 

Swapping a temporary subdomain from wordpress.com to wpcomstaging is not a 'feature' that is 'incompatible'. It's just part of the process. We should reword this to be more like:

> By uploading your theme your site's subdomain will change from abc123.wordpress.com to abc123.wpcomstaging.com. Questions?: Contact support for help.

**Comment by zdenys** [comment]
>Swapping a temporary subdomain from wordpress.com to wpcomstaging is not a 'feature' that is 'incompatible'. It's just part of the process. We should reword this

I completely agree on this. I was going through the site creation flow myself and can definitely see how the message is confusing for a new user. The same message should be also changed for when you are installing a plugin (not only a theme).

**Comment by gwwar** [comment]
Note that code is likely duplicated on all AT transfer points, so be sure to check Hosting/Plugin/Theme flows while testing this.

**Comment by gwwar** [comment]
Looking at code, we should get a list of warnings from the API of what things will change on a site. Usually this would be non-supported widgets/plugins, which is why we have the awkward client title of "This feature isn't (yet) compatible with `feature` and will be disabled:"

https://github.com/Automattic/wp-calypso/blob/07820b1729c35a8a72bb628ce81a05fce5924625/client/blocks/eligibility-warnings/warning-list.tsx#L91-L136

For the plural case, we can update Calypso copy to say something closer to: "Enabling X will make the following changes to your site:". (I'll ask editorial for help).

@jmdodd @aidvu I know the Atomic team has done a great job of whittling down the list of warnings and errors. How many warnings do we typically see per transfer? Would it be worth providing a singular text string option closer to what @rickybanister suggested from the API?

> By uploading your theme your site's subdomain will change from abc123.wordpress.com to abc123.wpcomstaging.com.

**Comment by aidvu** [comment]
Totally cool with your copy updates. I'd still keep the original format of having a top-level message, and then a list beneath (which I see is what you already did in the PR). :+1:
